### PR TITLE
4.frontend

### PR DESCRIPTION
**Pull request by ameyunaikgithub**

### CODE REVIEW

**Review by saumya665 — APPROVED** [review]

File: frontend/public/data.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello amey shoes
```
Comment (allopez7) [inline frontend/public/data.txt]: Get rid of this text file.

**Review by allopez7 — CHANGES_REQUESTED** [review]
Looks great other than the file that needs to be deleted.

File: frontend/express_html.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const fs =require("fs");
+const express =require("express");
+const app=express();
+app.listen(3614);
+app.use(express.static('public'));
+app.get('/form',(req,res)=>{
+    const name=req.query.username;
+    const comment=req.query.comment;
+    fs.writeFile('/home/amey/garageScript/frontend/public/data.txt',"hello"+" "+ name+" "+ comment);
```
Comment (allopez7) [inline frontend/express_html.js]: Why do you have hello and space. We're eventually going to create a chatroom. Name needs to be name: comment

**Review by allopez7 — APPROVED** [review]
Good Job!

**Pull request merged**